mix.exs

```diff
@@ -26,7 +26,7 @@ defmodule Scholar.MixProject do
     [
       {:ex_doc, "~> 0.29", only: :dev, runtime: false},
       {:nx, "~> 0.5.1"},
-      {:explorer, "~> 0.5.1", only: [:test, :dev]},
+      {:explorer, "~> 0.5.5", only: [:test, :dev]},
```

Comment:
Btw, in the future we want to avoid updating dependencies requirements. That's because doing so will force everyone to use latest Explorer, forcing them to update, and we should only do so if strictly necessary. If you want to update your own Explorer only, you can do `mix deps.update explorer`. :)